tests/functional/adapter/incremental_strategies/test_incremental_strategies.py

```diff
@@ -0,0 +1,125 @@
+import pytest
+
+from dbt.tests.util import run_dbt, check_relations_equal
+from dbt.tests.adapter.simple_seed.test_seed import SeedConfigBase
+from tests.functional.adapter.incremental_strategies.fixtures import *
+
+class TestIncrementalStrategies(SeedConfigBase):
+    @pytest.fixture(scope="class")
+    def schema(self):
+        return "incremental_strategies"
```

Comment:
I think you need to remove this, specifying a schema here can create weird impacts on testing